tests/test_types_leb128.py

```diff
@@ -0,0 +1,205 @@
+import io
+
+import pytest
+from dissect import cstruct
+
+
+def test_leb128_unsigned_read_EOF():
+    cs = cstruct.cstruct()
+
+    with pytest.raises(EOFError, match="EOF reached, while final LEB128 byte was not yet read"):
+        cs.uleb128(b"\x8b")
+
+
+def test_leb128_unsigned_read():
+    cs = cstruct.cstruct()
+
+    assert cs.uleb128(b"\x02") == 2
+    assert cs.uleb128(b"\x8b\x25") == 4747
+    assert cs.uleb128(b"\xc9\x8f\xb0\x06") == 13371337
+    assert cs.uleb128(b"\x7e") == 126
+    assert cs.uleb128(b"\xf5\x5a") == 11637
+    assert cs.uleb128(b"\xde\xd6\xcf\x7c") == 261352286
+
+
+def test_leb128_signed_read():
+    cs = cstruct.cstruct()
+
+    assert cs.ileb128(b"\x02") == 2
+    assert cs.ileb128(b"\x8b\x25") == 4747
+    assert cs.ileb128(b"\xc9\x8f\xb0\x06") == 13371337
+    assert cs.ileb128(b"\x7e") == -2
+    assert cs.ileb128(b"\xf5\x5a") == -4747
+    assert cs.ileb128(b"\xde\xd6\xcf\x7c") == -7083170
+
+
+def test_leb128_struct_unsigned(compiled):
```

Comment:
```suggestion
@pytest.mark.parametrize("compiled", [True, False])
def test_leb128_struct_unsigned(compiled):
```
Can you add this everywhere? v3 doesn't have the fixture for this yet.